CONTRIBUTING.md

```diff
@@ -0,0 +1,16 @@
+## Making changes to CitGM
+#### There are a few basic requirements for making changes to CitGM
+
+* Create an issue prior to submitting PR's as this can help speed up the process
+* Include tests for your code wherever possible
+* Include any documentation or manual changes where necessary
+* Ensure that `npm test` passes before submitting the PR
+* Please squash each logical change into one single commit
+* Follow the nodejs commit guidelines shown [here](https://github.com/nodejs/node/blob/master/CONTRIBUTING.md#step-3-commit)
+
+## Flaky and Skip Changes to lookup.json
+The process for adding flaky or skip tags to the lookup.json is reasonably simple. Make your changes put together a PR with ideally a link to the community CI failing and then it will be reviewed.
```

Comment:
This paragraph is mostly fluff. Also everything needs to be wrapped to <80chars (some editors do it automatically).